alembic/versions/33b920ecf3d6_add_incentives_apy_column.py

```diff
@@ -0,0 +1,27 @@
+"""add incentives_apy column
+
+Revision ID: 33b920ecf3d6
+Revises: fe1a6b4ecd8a
```

Comment:
```suggestion
Revises: 9563fd232d2f
```

Reply:
I fixed the migrations (there was one duplicate due to cherrypicking this branch a while ago) and the head migration is now `9563fd232d2f`